modules/bismark_methXtract.cfmod

```diff
@@ -64,13 +64,6 @@ The bismark_methXtract module runs the bismark_methylation_extractor script.
 This reads in a bisulfite read alignment results file produced by the Bismark bisulfite
 mapper and extracts the methylation information for individual cytosines in CpG, CHG 
 and CHH context.\n".
-# Specifying the pbat or single_cell parameters in a pipeline
```

Comment:
I guess we should probably also delete the commented out code below that this refers to. I forget why this isn't included in this module any more..?